src/ptvsd/_vendored/pydevd/_pydevd_bundle/pydevd_net_command_factory_json.py

```diff
@@ -191,9 +204,10 @@ def make_get_thread_stack_message(self, py_db, seq, thread_id, topmost_frame, fm
 
                     presentation_hint = None
                     if not getattr(frame, 'IS_PLUGIN_FRAME', False):  # Never filter out plugin frames!
+                        if py_db.is_files_filter_enabled and py_db.apply_files_filter(frame, original_filename, False):
```

Comment:
Note that the changes in test_exceptions_and_partial_exclude_rules are due to this change (the apply_files_filter handles both include/exclude rules as well as the project scope).